tests/raftstore_cases/test_conf_change.rs

```diff
@@ -776,5 +776,5 @@ fn test_learner_with_slow_snapshot() {
     // Transfer leader so that peer 3 can report to pd with `Peer` in memory.
     pd_client.transfer_leader(r1, new_peer(3, 3));
```

Comment:
According to the logs, peer 1 may send multiple snapshots to peer 3 when the environment is very slow because new entries in peer 1 are truncated when peer 3 has applied the snapshots.